MSBuild.lutconfig

```diff
@@ -0,0 +1,7 @@
+<LUTConfig Version="1.0">
+  <Repository />
+  <ParallelBuilds>true</ParallelBuilds>
+  <ParallelTestRuns>false</ParallelTestRuns>
+  <EnablePdbs>true</EnablePdbs>
```

Comment:
Just curious why this is true instead of false (the default). Do MSBuild tests depend on PDBs being generated?

If not, LUT's build would probably be a bit faster without with PDBs turned off. The only other downside of turning off PDBs is that you won't see clickable links for stack traces displayed in the LUT test window. But you could always debug the test reported as failing in LUT using the regular "Debug Test" command regardless of the above LUT setting.

Note that turning PDBs on will also currently lead to some minor weirdness in the LUT test window. The stack traces displayed there will contain file paths from LUT's shadow workspace folder - so when you click on them this will open the corresponding files as Miscellaneous Files (as opposed to opening the corresponding files from your solution). We plan to fix the LUT test window to translate paths and open the files from the solution soon - but in the meanwhile you may want to watch out for this :)

Reply:
We use https://github.com/shouldly/shouldly which has super pretty test assertions--but only if the PDB is available at test-assert time. When I artificially introduced some failures I noticed that we were falling back into the not-as-nice assertions.

Reply:
Ah interesting - I was not aware of assertion frameworks that need PDBs to be turned on. That makes sense.

cc @vritant24 @drognanar

Reply:
would that be something we should point out in the documentation?

Reply:
Yes. I don't think we need to call out specific frameworks - but we could provide examples of scenarios where the setting may need to be turned on (e.g., "Turn this on in case your tests / assertion frameworks depend on PDBs being turned on").